api/urls.py

```diff
@@ -125,4 +126,9 @@
 
 urlpatterns = [
     path("graphql/", csrf_exempt(FileUploadGraphQLView.as_view(graphiql=True))),
+    path(
+        "gdpr/v1/user/<str:uuid>/",
```

Comment:
:thinking: 

This is not super urgent but in the `tilavaraus/urls.py` this is imported as `from api.urls import urlpatterns as graphql` which is outdated after this update.

Reply:
oh good catch, I'll change the importing